Plugins/Public/base_plugin/Main.cpp

```diff
@@ -90,8 +90,19 @@ uint set_tick_time = 16;
 // How much damage do we heal per repair cycle?
 uint repair_per_repair_cycle = 60000;
 
-/// If the shield is up then damage to the base is changed by this multiplier.
-float set_shield_damage_multiplier = 0.03f;
+
+// set of configurable variables defining the diminishing returns on damage during POB siege
+// POB starts at base_shield_strength, then every 'threshold' of damage taken, 
+// shield goes up in absorption by the 'increment'
+float shield_reinforcement_threshold_flat = 1000000.0f;
+float shield_reinforcement_threshold_percent = 0.0f;
+float shield_reinforcement_increment = 0.0f;
+float base_shield_strength = 0.97f;
```

Comment:
why does this default to 0.97f instead of 1.0f? feels like something odd is going on here

Reply:
'base shield strength' is the starting damage reduction. I used the base 97%, along with 0.0f increments so that in case of no configuration applied at all, the siege mechanics will continue to work as it does now on live, which is static 97% the whole day. Values are overwritten by the config files, if present.

Reply:
Aingar clarified this over Discord - the 0.97f magic number is not entirely new, it's just that there was previously a `set_shield_damage_multiplier = 0.03f` now replaced with a definition of `0.97f` and `set_shield_damage_multiplier` references changed to `1 - shield_strength_multiplier`.